custom-webpack/universal-starter-master/src/app/home/home-view.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { TransferState } from '../../modules/transfer-state/transfer-state';
+
+@Component({
+  selector: 'home-view',
+  template: `<h3>{{ message }}</h3>`
+})
+export class HomeView implements OnInit {
+  public message: string;
+
+  constructor(private transferState: TransferState) {}
+
+  ngOnInit() {
+    this.message = this.transferState.get('message');
```

Comment:
transferState.get is be in the constructor  to replace the default value. this allows new values to happen via inputs in the component is nested

Reply:
Same as previous mention about the current webpack config'd project